Source/WebCore/Configurations/WebCoreTestSupport.xcconfig

```diff
@@ -58,11 +58,12 @@ PRODUCT_NAME = WebCoreTestSupport;
 EXECUTABLE_PREFIX = lib;
 EXPORTED_SYMBOLS_FILE = ;
 GCC_SYMBOLS_PRIVATE_EXTERN = YES;
-OTHER_LDFLAGS = $(ASAN_OTHER_LDFLAGS) $(ANGLE_LDFLAGS) -framework Metal $(SOURCE_VERSION_LDFLAGS);
-OTHER_LDFLAGS[sdk=macos*] = $(ASAN_OTHER_LDFLAGS) -framework AppKit -framework Metal $(ANGLE_LDFLAGS) $(LIBWEBRTC_LDFLAGS) $(SOURCE_VERSION_LDFLAGS);
+
+OTHER_LDFLAGS = $(WK_COMMON_OTHER_LDFLAGS) $(ANGLE_LDFLAGS) -framework Metal $(SOURCE_VERSION_LDFLAGS);
+OTHER_LDFLAGS[sdk=macos*] = $(WK_COMMON_OTHER_LDFLAGS) -framework AppKit -framework Metal $(ANGLE_LDFLAGS) $(LIBWEBRTC_LDFLAGS) $(SOURCE_VERSION_LDFLAGS);
```

Comment:
This is a specific target xcconfig.  I don't want to change its behavior in this patch.

I do agree this could be simplified, but this patch is already large, so I don't want to keep adding more changes to it, or it will increase the time it takes to land.  I prefer to break out changes like this into smaller patches.